openpype/settings/entities/schemas/projects_schema/schemas/schema_anatomy_imageio.json

```diff
@@ -3,7 +3,6 @@
     "key": "imageio",
     "label": "Color Management and Output Formats",
     "is_file": true,
-    "is_group": true,
```

Comment:
It's ok for me but I don't think it will change much. Anatomy is always fully overriden on project level.